packages/vm-config/decentral-itest-vaults-config.json

```diff
@@ -5,6 +5,8 @@
   "coreProposals": [
     "@agoric/builders/scripts/vats/init-core.js",
     "@agoric/builders/scripts/vats/init-network.js",
+    "@agoric/builders/scripts/vats/init-localchain.js",
+    "@agoric/builders/scripts/vats/init-transfer.js",
```

Comment:
In [5d4b6](https://github.com/Agoric/agoric-sdk/pull/9477/commits/5d4b6fdf855ed9ded306a104d72da1fd3b317ef9) `packages/vm-config/decentral-itest-vaults-config.json` was created to capture the orchestration proposals.

We might consider using that instead and removing `init-network.js` from the vaults config, but I don't have a strong opinion here.

Reply:
The reason for [5d4b6](https://github.com/Agoric/agoric-sdk/pull/9477/commits/5d4b6fdf855ed9ded306a104d72da1fd3b317ef9) was to have faster test for orchestration (@0xpatrickdev I edited above comment) and to leave itest-vaults with just what was needed to test vault factory.

I see @michaelfig wasn't on the review and had reason to expect those lines to remain. But I don't think we should restore to itest-vaults what isn't necessary to test vaults. Here's the comment describing what belongs in the config:
https://github.com/Agoric/agoric-sdk/blob/205e1a840b231c0dd85361e4fd2b678f8f37e051/packages/vm-config/decentral-itest-vaults-config.json#L2-L3

And here for itest-orchestration:
https://github.com/Agoric/agoric-sdk/blob/205e1a840b231c0dd85361e4fd2b678f8f37e051/packages/vm-config/decentral-itest-orchestration-config.json#L2

I don't see the need for a single config that launches both vaults and orchestration, but if there is then please define a new one with a comment on what its purpose is and what expectations the consumer should have for its maintenance and updates.